test/support/protocol_utils.rb

```diff
@@ -381,16 +338,12 @@ def req_set_breakpoints_on_dap
                           condition: condition
                         }
                       }
-        res = find_crt_dap_response
-        assert_dap_response :SetBreakpointsResponse, res
       }
     end
 
     def assert_disconnect_result
```

Comment:
I think that we can change this method name to something like `cleanup_reader` because it doesn't assert the result.

Reply:
done